yaml2sbml/yaml2sbml.py

```diff
@@ -65,9 +93,19 @@ def _parse_yaml_dict(yaml_dict) -> str:
         raise SystemExit('Could not create SBMLDocument object')
 
     model = document.createModel()
+
+    # remove file extension
+    if model_name.endswith('.xml') or model_name.endswith('.sbml'):
+        model_name = model_name[:-4]
```

Comment:
.sbml are 5 ...

Reply:
As above, using `pathlib.Path` should make this easy

Reply:
As the common maths joke says: There are three sorts of mathematicians: Those who can count, and those who don't -.- :D Thank you for catching this :+1: